webrtc.html

```diff
@@ -2848,12 +2817,24 @@
                     </ol>
                   </li>
                   <li>
-                    <p>Set <var>connection</var>'s <a>[[\IsClosed]]</a> slot to
-                    <code>true</code>.</p>
+                    <p>Set the <code><a data-link-for=
+                    "RTCDtlsTransport">state</a></code> of each of
+                    <var>connection</var>'s <code><a>RTCDtlsTransport</a></code>s
+                    to <code><a data-link-for=
+                    "RTCDtlsTransportState">"closed"</a></code>.</p>
                   </li>
                   <li>
-                    <p>Set <var>connection</var>'s <a>signaling state</a> to
-                    <code>"closed"</code>.</p>
+                    <p>Destroy <var>connection</var>'s <a>ICE Agent</a>,
+                    abruptly ending any active ICE processing and
+                    releasing any relevant resources (e.g. TURN
+                    permissions).</p>
+                  </li>
+                  <li>
+                    <p>Set the <code><a data-link-for=
+                    "RTCIceTransport">state</a></code> of each of
```

Comment:
[state](https://w3c.github.io/webrtc-pc/#dom-rtcicetransport-state) is a read-only attribute and cannot be set.

Reply:
I guess we need a state internal slot...

Reply:
Opened Issue https://github.com/w3c/webrtc-pc/issues/1574